indexing-service/src/main/java/io/druid/server/initialization/IndexerZkConfig.java

```diff
@@ -29,8 +30,8 @@
  */
 public class IndexerZkConfig
 {
-  @Inject
   @JsonIgnore
+  @JacksonInject
```

Comment:
Can we create a `@JsonCreator` constructor that sets all the properties and mark them as final in the class? That will make it more obvious what is injected and what not.